.gitignore

```diff
@@ -1,3 +1,17 @@
-/node_modules/
```

Comment:
Please revert these changes.

Reply:
I've reverted .gitignore, but `/test/tests/resources`  is added.